HTML+CSS+JS/Buttons/Icon-hover-expansion/index.html

```diff
@@ -0,0 +1,63 @@
+        </symbol>
+        <symbol xmlns="http://www.w3.org/2000/svg" viewBox="-21 -41 681.3351 681" id="snapchat">
+            <g stroke="none">
+                <path
+                    d="m323.402344 598.386719c-1.652344 0-6.996094 0-8.070313 0-37.84375 0-62.125-17.160157-85.613281-33.773438-16.21875-11.457031-31.527344-22.277343-49.558594-25.28125-8.800781-1.457031-17.558594-2.1875-26.007812-2.1875-15.257813 0-27.28125 2.351563-36.0625 4.070313-5.335938 1.046875-9.9375 1.9375-13.441406 1.9375-3.660157 0-7.605469-.789063-9.335938-6.671875-1.492188-5.105469-2.570312-10.027344-3.621094-14.808594-2.683594-12.28125-4.589844-19.839844-9.730468-20.628906-60.019532-9.269531-77.203126-21.921875-81.03125-30.886719-.539063-1.277344-.851563-2.570312-.9179692-3.835938-.1914068-3.449218 2.2343752-6.496093 5.6523432-7.046874 92.253907-15.191407 133.628907-109.492188 135.355469-113.492188.035157-.109375.082031-.21875.140625-.324219 5.652344-11.445312 6.75-21.382812 3.300782-29.527343-6.332032-14.910157-26.988282-21.46875-40.660157-25.808594-3.335937-1.0625-6.511719-2.058594-9.011719-3.058594-27.277343-10.78125-29.539062-21.84375-28.46875-27.480469 1.828126-9.621093 14.667969-16.320312 25.0625-16.320312 2.84375 0 5.347657.515625 7.457032 1.496093 12.28125 5.738282 23.328125 8.660157 32.878906 8.660157 13.183594 0 18.9375-5.542969 19.640625-6.269531-.339844-6.242188-.75-12.761719-1.179687-19.5-2.742188-43.613282-6.160157-97.824219 7.652343-128.753907 41.359375-92.726562 129.058594-99.945312 154.960938-99.945312.660156 0 11.359375-.113281 11.359375-.113281h1.53125c25.949218 0 113.851562 7.214843 155.238281 100.003906 13.800781 30.953125 10.378906 85.199218 7.628906 128.78125l-.109375 1.90625c-.386718 6.0625-.757812 11.960937-1.070312 17.613281.667968.675781 5.960937 5.769531 17.90625 6.226563h.007812c9.09375-.34375 19.527344-3.25 30.992188-8.605469 3.347656-1.570313 7.078125-1.898438 9.613281-1.898438 3.890625 0 7.816406.75 11.078125 2.109375l.203125.082032c9.257813 3.289062 15.335937 9.777343 15.46875 16.5625.121094 6.394531-4.757813 15.996093-28.6875 25.445312-2.480469.988281-5.65625 1.988281-9.011719 3.058594-13.691406 4.339844-34.328125 10.898437-40.660156 25.808594-3.449219 8.144531-2.351562 18.070312 3.300781 29.515624.046875.117188.101563.21875.152344.335938 1.714844 4 43.042969 98.273438 135.347656 113.484375 3.40625.558594 5.847657 3.605469 5.660157 7.046875-.078126 1.289062-.402344 2.589844-.949219 3.859375-3.8125 8.898437-20.972657 21.535156-81 30.800781-4.914063.757813-6.820313 7.152344-9.742188 20.558594-1.066406 4.871094-2.140625 9.664062-3.621093 14.703125-1.28125 4.375-4.003907 6.417969-8.597657 6.417969h-.738281c-3.183594 0-7.699219-.578125-13.433594-1.703125-10.164062-1.984375-21.558594-3.808594-36.0625-3.808594-8.46875 0-17.226562.730469-26.035156 2.1875-18.003906 3-33.300781 13.796875-49.492188 25.242187-23.523437 16.652344-47.808593 33.816407-85.667968 33.816407" />
```

Comment:
I am not sure about these tags and code words. I tried running the code without these tags and it was working fine. I feel that these are being used to create symbols for the different social media logos being used on the website. 

@Pranav016 could you please have a second look into it and see if we should merge it.

Reply:
Hey @theBeginner86 I haven't written this part of code I found on web for icons and just copied the code for that icon